File: programme/celsius-to-fahrenheit1/celsius-to-fahrenheit1.c
//C# program to convert the temperature from Celsius to Fahrenheit

#include <stdio.h>

void main()
{
  float c,f;
  printf("Input ( C ) : ");
  scanf("%f", &c);
  f = ((9 * c) / 5) + 32;
  printf("Output ( F )  : %f\n", f);
}

File: programme/celsius-to-fahrenheit1/celsius-to-fahrenheit1.java
/** Java program to convert celsius to fahrenheit */

import java.util.Scanner;
class Celsius_to_Fahrenheit 
{
 public static void main(String[] args)
{
 float cel, far;

Scanner s=new Scanner(System.in);
System.out.println("Input ( C ) :");
cel=s.nextInt();

far = cel * 9/5 + 32;

System.out.println("Output ( F ): "+far);
}
}